src/store/modules/nft.js

```diff
@@ -456,7 +469,12 @@ const actions = {
     }
     return info;
   },
-  parseAndStoreNFTClassMetadata({ commit }, { classId, classData = {} }) {
+  parseAndStoreNFTClassMetadata(
+    { commit, getters },
+    { classId, classData = {} }
+  ) {
+    // Do not overwrite aggregated metadata
+    if (getters.getNFTClassMetadataById(classId)) return;
```

Comment:
to avoid breaking change, we can
```suggestion
    if (getters.getNFTClassMetadataById(classId)) return getters.getNFTClassMetadataById(classId);
```